src/_guides/language/effective-dart/usage_migrated.md

```diff
@@ -1110,6 +1110,11 @@ class ProfileMark {
 }
 {% endprettify %}
 
+One common reason that fields are *not* initialized at the declaration is
+because they need to reference `this` or call methods on the containing
+instance. However, if the field is marked `late`, then the initializer *can*
+access `this`.
```

Comment:
This can be read as if fields always need access to `this`. How about something like this (which I think you could improve):

```suggestion
Some fields can't be initialized at declaration because they need to reference
`this` — to use other fields or call methods, for example. However, if the
field is marked `late`, then the initializer *can* access `this`.
```